client/src/http/httpClient.js

```diff
@@ -0,0 +1,40 @@
+import { createClient } from './index.js';
+import { authService } from '../services/authService.js';
+import { accessTokenService } from '../services/accessTokenService.js';
+
+export const httpClient = createClient();
+
+httpClient.interceptors.request.use(onRequest);
+httpClient.interceptors.response.use(onResponseSuccess, onResponseError);
+
+function onRequest(request) {
+  const accessToken = localStorage.getItem('accessToken');
+
+  if (accessToken) {
+    request.headers['Authorization'] = `Bearer ${accessToken}`;
+  }
+
+  return request;
+}
+
+function onResponseSuccess(res) {
+  return res.data;
+}
+
+async function onResponseError(error) {
+  const originalRequest = error.config;
+
+  if (error.response.status !== 401) {
+    throw error;
+  }
```

Comment:
This condition checks if the error response status is not 401. It might be more robust to also check if `error.response` is defined to avoid potential runtime errors if `error.response` is undefined.